splunk-quarkus/init_ephemeral_cluster.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env
+
+# See step 7 of this link: https://clouddot.pages.redhat.com/docs/dev/getting-started/ephemeral/onboarding.html
+echo "Logging into Openshift cluster"
+sh ~/ephemeral-login.sh
```

Comment:
Where can we grab this script?

Reply:
Please, refer to step 7 of this doc: https://consoledot.pages.redhat.com/docs/dev/getting-started/ephemeral/onboarding.html

Reply:
Oh :see_no_evil:. I'd rather replace it with the oc login command, as the token changes and the script above is polluting home dir. I'll try to update their doc.

Reply:
Can we instead check the login status and ask to log in in case the session has timed out?
```
$ oc whoami --show-server
https://api.c-rh-c-eph.8p0c.p1.openshiftapps.com:6443
```
this command can be used to check the login status and verify the server.